internal/woc_client/woc_client.go

```diff
@@ -84,7 +94,7 @@ func (w *WocClient) GetUTXOsWithRetries(ctx context.Context, lockingScript *scri
 	operation := func() (sdkTx.UTXOs, error) {
 		wocUtxos, err := w.GetUTXOs(ctx, lockingScript, address)
 		if err != nil {
-			return nil, fmt.Errorf("failed to get utxos from WoC: %v", err)
+			return nil, errors.Join(ErrWOCFailedToGetUTXOs, fmt.Errorf("error: %v", err))
```

Comment:
This msg here is pointless, we can just do:
```suggestion
			return nil, errors.Join(ErrWOCFailedToGetUTXOs, err)
```

Reply:
Done